### PR TITLE
Add transparency dev summit

### PR DESCRIPTION
**Pull request by tracymiranda**

### CODE REVIEW

File: content/_index.html [hunk 1]
```diff
@@ -42,6 +42,35 @@ <h1 class="headline1 font-medium">
...
+  <a id="get-started"></a>
+  <div class="glue-grid gpt-6">
+    <div class="col-span-4 sm:col-span-10 sm:col-start-2 lg:col-span-4 lg:col-start-2">
+      <h2 class="headline2 gmb-4">
+        Join the Community
+      </h2>
+    </div>
+    <div class="col-span-4 sm:col-span-10 sm:col-start-2 lg:col-span-6">
+      <p class="gmb-2 headline5 intro-para">
+        Welcome to Transparency.dev, a community dedicated to promoting transparency across various ecosystems and pioneering its integration into new domains.
```
Comment (AlCutter) [inline content/_index.html:55]: nit: is it worth expanding to "application domains" given the history of CT?

File: content/application/transparency-dev-summit.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      <img src="{{< rel "/images/transparency-dev-summit/tdev_summit_banner.svg" >}}" class="w-full" />
+    </div>
+  </div>
+</div>
+
+<section class="glue-page">
+  <div class="glue-grid gmb-7">
+    <div class="col-span-4 sm:col-span-6 sm:col-start-4">
+      <p class="headline5 gmb-2">
+        Join us for Transparency.dev Summit at Google Academy London on October 9-11, 2024.
```
Comment (AlCutter) [inline content/application/transparency-dev-summit.html:27]: "the Transparen...", or was it intentional?

File: content/application/transparency-dev-summit.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+url: /transparency-dev-summit
```
Comment (roger2hk) [inline content/application/transparency-dev-summit.html:2]: "transparency" and "dev" are duplicated in the actual URL (i.e. https://transparency.dev/transparency-dev-summit).

File: content/_index.html [hunk 4]
```diff
@@ -42,6 +42,35 @@ <h1 class="headline1 font-medium">
   </div>
 </section>
 
+<section class="glue-page gpb-8">
+  <a id="get-started"></a>
```
Comment (roger2hk) [inline content/_index.html:46]: Can this `<a>` element with empty content be removed?

File: content/_index.html (hunk 1 above)
Comment (tracymiranda) [reply content/_index.html:55]: +1 good should, will fix

File: content/_index.html (hunk 4 above)
Comment (tracymiranda) [reply content/_index.html:46]: +1, good spot

File: content/application/transparency-dev-summit.html (hunk 3 above)
Comment (tracymiranda) [reply content/application/transparency-dev-summit.html]: Fixed

**Comment by tracymiranda** [comment]
@AlCutter @roger2hk addressed your feedback, please see screenshots above.

**Review by mhutchinson — APPROVED** [review]

**Pull request merged**